### ISSUE TITLE
PathPrefixStrip Is not working as intended

### ISSUE THREAD

**Issue by ray-matos-mpf** [containous/traefik#2722]
Given the follow compose file

```yml
  apiv1:
    image: wouterla/whoami
    networks:
    - proxy
    environment:
      - WHOAMIserviceName=apiv1
    deploy:
      replicas: 2
      labels:
      - traefik.frontend.rule=PathPrefixStrip:/api/v1
      - traefik.priority = 1
      - traefik.docker.network=proxy
      - traefik.port=8000
      - traefik.enable=true
  apiv2:
    image: wouterla/whoami
    networks:
    - proxy
    environment:
      - WHOAMIserviceName=apiv2
    deploy:
      replicas: 2
      labels:
      - traefik.frontend.rule=PathPrefixStrip:/api/v2;/api
      - traefik.priority = 2
      - traefik.docker.network=proxy
      - traefik.port=8000
      - traefik.enable=true
```

Expected
 - /api and /api/v2 hits apiv2 backend 
 - /api/v1 hits the apiv1 backend

Actual
 - /api returns 404
 - /api/v2 hits apiv2 backend 
 - /api/v1 hits apiv1 backend

I also tried 
```traefik.frontend.rule=PathPrefixStrip:/api/v2,/api```
But this resolves all routes to apiv2

Thanks in advance

**Comment by nmengin** [comment]
Hello @ray-matos-mpf .

Many thanks for your interest in the project.

Can you send us the version of Træfik you use?

Thanks in advance.

**Comment by nmengin** [comment]
Hello @ray-matos-mpf ,

We analyzed deeply your issue and it appears it contains mistakes which explain why it does not work as expected.

- The labels `traefik.priority` have to be renamed into `traefik.frontend.priority`,
- The spaces in the right part of the labels `traefik.frontend.priority` have to be removed
- The `;` in the second rule has to be replaced by a `,`
- The priority are not specified correctly, you can refer to [the documentation](https://docs.traefik.io/basics/#priorities) for more details.

The configuration described below contains all the modification and works well : 

```yaml
  apiv1:
    image: wouterla/whoami
    networks:
    - proxy
    environment:
      - WHOAMIserviceName=apiv1
    deploy:
      replicas: 2
      labels:
      - traefik.frontend.rule=PathPrefixStrip:/api/v1
      - traefik.frontend.priority=5
      - traefik.docker.network=proxy
      - traefik.port=8000
      - traefik.enable=true
  apiv2:
    image: wouterla/whoami
    networks:
    - proxy
    environment:
      - WHOAMIserviceName=apiv2
    deploy:
      replicas: 2
      labels:
      - traefik.frontend.rule=PathPrefixStrip:/api/v2,/api
      - traefik.frontend.priority=1
      - traefik.docker.network=proxy
      - traefik.port=8000
      - traefik.enable=true
```

The documentation about priorities will be improved in a Pull Request to propose configuration like yours.

**Comment by nmengin** [comment]
Documentation fixed in #2984.
Thanks @jbdoumenjou.

I close the issue
